<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Virtual Machine - 标签 - Wenyi&#39;s Blog</title>
    <link>https://blog.wenyi.org/tags/virtual-machine/</link>
    <description>Wenyi&#39;s Blog</description>
    <generator>Hugo 0.139.3 &amp; FixIt v0.3.16</generator>
    <language>zh-CN</language>
    <lastBuildDate>Tue, 29 Mar 2022 13:30:00 +0000</lastBuildDate>
    <atom:link href="https://blog.wenyi.org/tags/virtual-machine/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>解决 VMware 虚拟机运行时 “内部错误” 问题</title>
      <link>https://blog.wenyi.org/posts/vmware-internal-errors/</link>
      <pubDate>Tue, 29 Mar 2022 13:30:00 +0000</pubDate>
      <guid>https://blog.wenyi.org/posts/vmware-internal-errors/</guid>
      <category domain="https://blog.wenyi.org/categories/%E6%95%99%E7%A8%8B/">教程</category>
      <description>&lt;h2 id=&#34;起因&#34; class=&#34;heading-element&#34;&gt;&lt;span&gt;起因&lt;/span&gt;&#xD;&#xA;  &lt;a href=&#34;#%e8%b5%b7%e5%9b%a0&#34; class=&#34;heading-mark&#34;&gt;&#xD;&#xA;    &lt;svg class=&#34;octicon octicon-link&#34; viewBox=&#34;0 0 16 16&#34; version=&#34;1.1&#34; width=&#34;16&#34; height=&#34;16&#34; aria-hidden=&#34;true&#34;&gt;&lt;path d=&#34;m7.775 3.275 1.25-1.25a3.5 3.5 0 1 1 4.95 4.95l-2.5 2.5a3.5 3.5 0 0 1-4.95 0 .751.751 0 0 1 .018-1.042.751.751 0 0 1 1.042-.018 1.998 1.998 0 0 0 2.83 0l2.5-2.5a2.002 2.002 0 0 0-2.83-2.83l-1.25 1.25a.751.751 0 0 1-1.042-.018.751.751 0 0 1-.018-1.042Zm-4.69 9.64a1.998 1.998 0 0 0 2.83 0l1.25-1.25a.751.751 0 0 1 1.042.018.751.751 0 0 1 .018 1.042l-1.25 1.25a3.5 3.5 0 1 1-4.95-4.95l2.5-2.5a3.5 3.5 0 0 1 4.95 0 .751.751 0 0 1-.018 1.042.751.751 0 0 1-1.042.018 1.998 1.998 0 0 0-2.83 0l-2.5 2.5a1.998 1.998 0 0 0 0 2.83Z&#34;&gt;&lt;/path&gt;&lt;/svg&gt;&#xD;&#xA;  &lt;/a&gt;&#xD;&#xA;&lt;/h2&gt;&lt;p&gt;最近在使用 VMware 虚拟机的时候，遇到了一个问题。问题出现在虚拟系统重新启动后，提示“内部错误”。&#xA;&lt;img loading=&#34;lazy&#34; src=&#34;https://blog.wenyi.org/posts/vmware-internal-errors/1.webp&#34; alt=&#34;内部错误&#34; srcset=&#34;https://blog.wenyi.org/posts/vmware-internal-errors/1.webp?size=small, https://blog.wenyi.org/posts/vmware-internal-errors/1.webp?size=medium 1.5x, https://blog.wenyi.org/posts/vmware-internal-errors/1.webp?size=large 2x&#34; data-title=&#34;内部错误&#34; style=&#34;--width: 373px;--aspect-ratio: 373 / 199;background: url(/images/loading.min.svg) no-repeat center;&#34; onload=&#34;this.title=this.dataset.title;for(const i of [&#39;style&#39;, &#39;data-title&#39;,&#39;onerror&#39;,&#39;onload&#39;]){this.removeAttribute(i);}this.dataset.lazyloaded=&#39;&#39;;&#34; onerror=&#34;this.title=this.dataset.title;for(const i of [&#39;style&#39;, &#39;data-title&#39;,&#39;onerror&#39;,&#39;onload&#39;]){this.removeAttribute(i);}&#34;/&gt;&lt;/p&gt;</description>
    </item>
  </channel>
</rss>
